### PR TITLE
MAM-3757-dont-show-thumbnails-for-likes-and-reposts-in-activity

### PR DESCRIPTION
**Pull request by bnolens**
[MAM-3757 : Don’t show thumbnails for likes and reposts in activity.](https://linear.app/theblvd/issue/MAM-3757/dont-show-thumbnails-for-likes-and-reposts-in-activity)

... also, don't show quote posts and link cards (https://linear.app/theblvd/issue/MAM-3757/dont-show-thumbnails-for-likes-and-reposts-in-activity)

### CODE REVIEW

File: Mammoth/Models/PostCardModel.swift [hunk 1]
```diff
@@ -118,6 +118,15 @@ final class PostCardModel {
         case singleVideo
         case carousel
         case none
+        
+        var displayName: String? {
+            switch self {
+            case .singleImage: return "image"
+            case .singleVideo: return "video"
+            case .carousel: return "carousel"
```
Comment (rileyhCode) [inline Mammoth/Models/PostCardModel.swift:126]: Consider changing this to "images" in the context of it being displayed to the user:
     "Joe liked your images" (vs. "Joe liked your carousel")?

**Review by rileyhCode — APPROVED** [review]
Pls review the one comment, then merge as you see fit.

**Comment by rileyhCode** [comment]
Once this change is in, We need to re-apply the change for MAM-3752.

File: Mammoth/Models/PostCardModel.swift (hunk 1 above)
Comment (bnolens) [reply Mammoth/Models/PostCardModel.swift:126]: I believe I'm following Justin's recommendation here. I'll ask again and will create a separate PR if needed to change it.

**Comment by bnolens** [comment]
Not only thumbnail media needs to be hidden, but also quote posts and link cards. Just added the change to this branch since it's related. https://linear.app/theblvd/issue/MAM-3757/dont-show-thumbnails-for-likes-and-reposts-in-activity)

**Review by rileyhCode — APPROVED** [review]
LGTM

**Pull request merged**